test/e2e/e2e_setup_test.go

```diff
@@ -35,13 +34,13 @@ import (
 )
 
 const (
-	photonDenom    = "photon"
-	initBalanceStr = "110000000000uumee,100000000000photon"
-	minGasPrice    = "0.00001"
-	gaiaChainID    = "test-gaia-chain"
+	photonDenom    string = "photon"
+	initBalanceStr string = "110000000000uumee,100000000000photon"
+	minGasPrice    string = "0.00001"
+	gaiaChainID    string = "test-gaia-chain"
 
-	ethChainID uint = 15
-	ethMinerPK      = "0xb1bab011e03a9862664706fc3bbaa1b16651528e5f0e7fbfcbfdd8be302a13e7"
+	ethChainID uint   = 15
+	ethMinerPK string = "0xb1bab011e03a9862664706fc3bbaa1b16651528e5f0e7fbfcbfdd8be302a13e7"
```

Comment:
we don't need to add types there, let's keep amount of changes to minimum

Reply:
done at https://github.com/umee-network/peggo/pull/343/commits/c8e67c98cf8e0bc4f1bae59b5d07653af78d56d3